src/index.html

```diff
@@ -9,6 +9,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="navigation__container">
+        <a href="#home" class="navigation__logo">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a href="#" class=" is-active navigation__link">
+              Apple
+            </a>
+          </li>
+          <li class="navigation__item">
```

Comment:
don`t forget to add the empty line between sibling tags

Reply:
![image](https://user-images.githubusercontent.com/12642122/218419204-9602da6d-4217-4d4c-a646-271563ea0637.png)